job-server/src/main/scala/spark/jobserver/io/FileCacher.scala

```diff
@@ -50,10 +50,10 @@ trait FileCacher {
       tempOutFileName, targetFullBinaryName: Any)
 
     val tempFile = new File(rootDir, tempOutFileName)
-    if( ! tempFile.renameTo(new File(rootDir, targetFullBinaryName))) {
+    if(!tempFile.renameTo(new File(rootDir, targetFullBinaryName))) {
```

Comment:
Can you re-base with master as I have pushed some scalastyle config to detect such style issues? Also it will be great if you could squash the commits into one.